aws/logs_monitoring/lambda_function.py

```diff
@@ -19,6 +19,21 @@
 
 import boto3
 
+try:
+    from datadog_lambda.wrapper import datadog_lambda_wrapper
+    from datadog_lambda.metric import lambda_metric
+    DD_FORWARD_METRIC = True
+except ImportError:
+    # Datadog Lambda layer is required to forward metrics
+    DD_FORWARD_METRIC = False
+
+
+# Set this variable to `False` to disable log forwarding.
+# E.g., you only want to forward metrics from logs, but not logs.
+DD_FORWARD_LOG = os.getenv("DD_FORWARD_LOG", default="true").lower() == "true"
+if not DD_FORWARD_LOG:
+    print("Log forwarding is disabled")
```

Comment:
Do we want to print this for every single lambda function run?

Reply:
I was printing it for every single invocation, but I realized that it would be way too verbose for metric-only customers (the primary use case). It would also contribute 33% (function start, this line, and function stop) of the cost of CW logs for the forwarder lambda itself. I also felt that it's quite easy to notice this environment variable in AWS console, if a customer is wondering why logs are not being forwarded.

Do you think it should be logged for every run?

Reply:
I think we should completely remove this line